command/operator_raft_join.go

```diff
@@ -13,10 +13,11 @@ var _ cli.Command = (*OperatorRaftJoinCommand)(nil)
 var _ cli.CommandAutocomplete = (*OperatorRaftJoinCommand)(nil)
 
 type OperatorRaftJoinCommand struct {
-	flagRaftRetry        bool
+	flagRetry            bool
```

Comment:
If the base command needs a `flagRetry` for some reason, we don't want to step on it through this command's flag? Maybe I am overthinking, but..